app/models/admin/account_action.rb

```diff
@@ -92,6 +92,10 @@ def process_strike!
       text: text_for_warning,
       status_ids: status_ids
     )
+
+    # A log entry is only interesting if the warning contains
+    # custom text from someone. Otherwise it's just noise.
+    log_action(:create, @warning) if @warning.text.present?
```

Comment:
This does restore the previous logic, but in the previous logic, warnings were mostly separate from other moderator actions, whereas now a moderator action and a warning almost always happen at the same time. Would it make more sense to log if the action is a no-op? So "just" a warning?

Reply:
Why not.